deploy/ocp-templates/heketi-template.yaml

```diff
@@ -11,6 +11,16 @@ metadata:
 labels:
   template: heketi
 objects:
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: heketi-db-backup
+    labels:
+      glusterfs: heketi-db
+      heketi: db
```

Comment:
Could be helpful to search nothing in particular. Can remove not anything important, Do you want me to remove?